rust_snuba/src/consumer.rs

```diff
@@ -152,6 +152,8 @@ pub fn consumer_impl(
                     max_invalid_ratio: Some(0.01),
                     max_consecutive_count: Some(1000),
                 },
+                // TODO: What should this actually be set to?
+                None,
```

Comment:
`None` is fine, we set that here as well: https://github.com/getsentry/snuba/blob/5a597e617339018a5d07ee0c9f893e0e05a7fde5/snuba/consumers/consumer_builder.py#L402